internals/webpack/webpack.prod.js

```diff
@@ -39,13 +39,6 @@ module.exports = require('./webpack.base.babel')({
     // Merge all duplicate modules
     new webpack.optimize.DedupePlugin(),
 
-    // Minify and optimize the JavaScript
-    new webpack.optimize.UglifyJsPlugin({
```

Comment:
Sorry, ignore, didn't read OP. "removed prod jsUglify as this is automatically done with -p arg"